### ISSUE TITLE
`participants.json` is overwritten on the second pass

### ISSUE THREAD

**Issue by gauvinalexandre** [jmtyszka/bidskit#128]
When running bidskit for the second time, the file `participants.json` is overwritten, event if I've set it myself. Then, bids-validator obviously complains as the columns of the `participants.tsv` are not described...

Should'nt bidskit detect if a pre-existing file exist before overwriting it?

**Comment by jmtyszka** [comment]
Good catch and yes, bidskit shouldn't be overwriting user edited files like these top-level JSONs. Working on a fix now.
